joystick_commander/src/node.c

```diff
@@ -82,61 +72,14 @@ typedef struct
 // *****************************************************
 
 /**
- * @brief Enable controls parameter identifier.
- *
- */
-static const ps_parameter_id ENABLE_CONTROLS_PARAMETER_ID = 4011;
-
-
-/**
- * @brief Platform brake command message type name.
- *
- */
-static const char BRAKE_COMMAND_MSG_NAME[] = "ps_platform_brake_command_msg";
-
-
-/**
- * @brief Platform throttle command message type name.
- *
- */
-static const char THROTTLE_COMMAND_MSG_NAME[] = "ps_platform_throttle_command_msg";
-
-
-/**
- * @brief Platform steering command message type name.
- *
- */
-static const char STEERING_COMMAND_MSG_NAME[] = "ps_platform_steering_command_msg";
-
-
-/**
- * @brief Gear position command message type name.
- *
- */
-static const char GEAR_POSITION_COMMAND_MSG_NAME[] = "ps_platform_gear_command_msg";
-
-
-/**
- * @brief Turn signal command message type name.
- *
- */
-static const char TURN_SIGNAL_COMMAND_MSG_NAME[] = "ps_platform_turn_signal_command_msg";
-
-
-/**
- * @brief Turn signal command message type name.
- *
- */
-static const char PARAMETERS_MSG_NAME[] = "ps_parameters_msg";
-
-
-/**
  * @brief Warning string.
  *
  */
 static const char WARNING_STRING[] =
 "\nWARNING: example is built for "
 "the Joystick device: Logitech Gamepad F310\n"
+"Back mode switch: 'X' setting\n"
```

Comment:
there is a switch on the back of the controller, it has two positions one being labelled 'X'